packages/flame_test/pubspec.yaml

```diff
@@ -2,13 +2,15 @@ name: flame_test
 description: A package with classes to help testing applications using Flame
 version: 1.1.0
 homepage: https://github.com/flame-engine/flame/tree/main/packages/flame_test
+publish_to: none
 
 environment:
   sdk: ">=2.14.0 <3.0.0"
   flutter: ">=2.5.0"
 
 dependencies:
-  flame: ^1.0.0
+  flame:
+    path: ../flame
```

Comment:
These can be set back to what they were before once this PR is rebased, now when #1354 is merged.